skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SkateService.kt

```diff
@@ -46,7 +51,11 @@ class SkateProjectServiceImpl(private val project: Project) : SkateProjectServic
           stripeTitle = Supplier { "What's New in Slack!" }
           anchor = ToolWindowAnchor.RIGHT
         }
-      WhatsNewPanelFactory().createToolWindowContent(toolWindow, changeLogString)
+
+      val parentDisposable = Disposer.newDisposable() // Creating a new Disposable
```

Comment:
This comment is a bit redundant. Instead, let's say what it's for.

Reply:
Sure, I added the comment there: ef81fb1e30983db32d66a35bdaf08df2bd0a5e5a